client/beefy/src/lib.rs

```diff
@@ -78,22 +83,34 @@ pub fn beefy_peers_set_config(
 	cfg
 }
 
+/// Produce a BEEFY block import object and a link half for tying it to the client
```

Comment:
This is misleading. The gadget will be appending justifications as well and notifications will not be triggered here.